tests/StatsdClient.Tests/DurationTools.cs

```diff
@@ -0,0 +1,24 @@
+using System;
+using NUnit.Framework;
+
+namespace Tests
+{
+    static class DurationTools
```

Comment:
I usually avoid using time measurements and `sleep` in unit test as it makes unit tests flaky. When I wrote the unit tests I did not see a clear way to avoid it but after a second thought, I can simply mock the sleep function.

Reply:
After a second thought I removed this class.